CONTRIBUTING.md

```diff
@@ -72,6 +72,28 @@ opentelemetry-collector-builder \
 2021-05-24T16:32:37.326+0200    INFO    builder/main.go:113     Compiled        {"binary": "./cmd/otelcol-sumo-linux_arm64"}
 ```
 
+### How to build packages (Windows MSI)
+
+1. Install Visual Studio 2019 or newer with the  `.NET desktop development`
+   workload selected in the installer.
+
+1. Open `cmd` and add msbuild to the user PATH. The path may differ depending on
```

Comment:
Perhaps use the "Developer Command Prompt for VS 2019/2022" instead of using the regular "cmd" and adding MSBuild to path?